keyboards/gh60/revc/revc.h

```diff
@@ -12,17 +12,17 @@
  *   B2 Capslock LED
  *   B0 not connected
  */
-inline void gh60_caps_led_on(void)      { setPinOutput(B2); writePinLow(B2); }
+inline void gh60_caps_led_on(void)      { setPinOutput(B2); writePinLow(B2); 
```

Comment:
```suggestion
inline void gh60_caps_led_on(void)      { setPinOutput(B2); writePinLow(B2); }
```

Reply:
That's what I get for doing this on the train right before my stop.